mage/query-modules/python/import-util.md

```diff
@@ -139,6 +139,83 @@ where `path` is the path to a local JSON file that will be created inside the
 
 </Tabs>
 
+### `graphml(path, config)`
+
+#### Input:
+
+* `path: string` ➡ path to the graphML file that is being imported.
+* `config: Map (default={})` ➡ configuration parameters explained below.
+
+#### Parameters:
+
+| Name 	             | Type   | Default	| Description 	                                                |
+|-	                 |-	      |-	      |-	                                                            |
+| readLabels          | Bool 	| False	  | Create node labels based on the value of the property whose key is "labels". 	|
```

Comment:
```suggestion
| readLabels          | Bool 	| False	  | Create node labels by using the value of the `labels` property. |
```

1. nodes ain't people
2. the original sentence is very complicated... is mine ok?